bin/test_balrogworker.py

```diff
@@ -226,16 +226,19 @@ def test_create_submitter_nightly_metadata(self):
         submitter, release = bw.create_submitter(nightly_manifest[0], args)
 
         exp = {
-            'partialInfo': [{
-                'url': 'https://queue.taskcluster.net/v1/task/e2q3BKuhRxqtcB6FVCbKfg/artifacts/public/env/Firefox-mozilla-central-50.0a1-linux-en-US-20160721030216-20160722030235.partial.mar',
-                'hash': u'bceca092b0334a868f86c2b7c6562e20044ccef4682e04065bea09e137d7e05ec4b157268703d9db2cbddfae3112edf2f0a3b04675a0ee59f50601412e352a39',
-                'from_buildid': u'20160721030216', 'size': 9776256}], 'hashFunction': 'sha512',
-            'extVersion': u'50.0a1', 'buildID': u'20160722030235', 'appVersion': u'50.0a1',
-            'productName': u'Firefox', 'platform': u'linux', 'branch': u'mozilla-central', 'locale': u'en-US',
+            'platform': "android-api-15",
+            'buildID': "20161107171219",
+            'productName': "Fennec",
```

Comment:
What do we gain from (at least with touched lines here) removing 'Firefox' in one set and making both 'Fennec' ?